app/services/questionnaire_responses/monthly_screening_reports.rb

```diff
@@ -3,17 +3,17 @@ class QuestionnaireResponses::MonthlyScreeningReports
 
   def initialize(date = 1.month.ago)
     @month_date = date.beginning_of_month
-    @facility_reports = monthly_followup_and_registration
+    @facility_ids = Facility.select("id").pluck(:id)
```

Comment:
We moved from iterating over `facility_reports` to `facilities` because it's possible mat-view refreshes wouldn't have completed and generated reports for the advanced month for which we're initializing dynamic forms.